approval_polls/templates/approval_polls/embed_instructions.html

```diff
@@ -1,7 +1,43 @@
 {% extends 'base.html' %}
 
 {% block content %}
-  Put this code in your site:
-  <textarea rows=3 cols=100><iframe src='{{ link }}' width='250'></iframe></textarea>
+
+<script type='text/javascript'>
```

Comment:
It's best to separate the JavaScript code from the HTML template by putting it in the [static](https://github.com/electology/approval_frame/tree/master/approval_polls/static/approval_polls) directory. You can then include the JavaScript file using the `staticfiles` app similar to how `create.js` is included [here](https://github.com/electology/approval_frame/blob/master/approval_polls/templates/approval_polls/create.html#L7)